pkg/sink/interceptors_test.go

```diff
@@ -88,7 +88,7 @@ func TestCreateOutgoingRequest(t *testing.T) {
 		"foo":       "bar",
 	})
 
-	req, _ := http.NewRequest(http.MethodPost, "http://event.listener.url", ioutil.NopCloser(bytes.NewBuffer(reqBody)))
+	req := httptest.NewRequest(http.MethodPost, "http://event.listener.url", ioutil.NopCloser(bytes.NewBuffer(reqBody)))
```

Comment:
"httptest.NewRequest returns a new incoming server Request, suitable for passing to an http.Handler for testing."